lib/propolis/src/hw/virtio/queue.rs

```diff
@@ -752,7 +752,6 @@ pub mod migrate {
     }
 }
 
-#[cfg(feature = "falcon")]
```

Comment:
I suspect this will grumble about unused code without the guard?

Reply:
Yep, fixed.